pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumers.java

```diff
@@ -236,8 +236,8 @@ protected synchronized boolean trySendMessagesToConsumers(ReadType readType, Lis
             Consumer consumer = current.getKey();
             List<Entry> entriesWithSameKey = current.getValue();
```

Comment:
```suggestion
            Consumer consumer = current.getKey();
            assert consumer != null; // checked when added to groupedEntries
            List<Entry> entriesWithSameKey = current.getValue();
```

... for better IDE lint.